packages/ui/components/avatar/AvatarGroup.tsx

```diff
@@ -36,7 +36,6 @@ export const AvatarGroup = function AvatarGroup(props: AvatarGroupProps) {
             imageSrc={item.image}
             title={item.title}
             alt={item.alt || ""}
-            accepted={props.accepted}
```

Comment:
What's the reason to remove it?

Also, if we are removing it we should cleanup `AvatarGroupProps` type as well.